source/connect.txt

```diff
@@ -6,13 +6,98 @@ Connect to Your MongoDB Deployment
 
 .. default-domain:: mongodb
 
+.. facet::
+   :name: genre
+   :values: tutorial
+
+.. meta:: 
+   :description: Connect to your MongoDB deployment with the MongoDB for IntelliJ Plugin.
+
 .. contents:: On this page
    :local:
    :backlinks: none
    :depth: 1
    :class: singlecol
 
-Content on this page will be addressed in DOCSP-44809.
+This task page shows you how to connect to your MongoDB deployment using
+the {+intellij-full+}.
+
+.. note:: MongoDB Atlas
+
+   If you need to create a MongoDB host, consider using 
+   `MongoDB Atlas <https://www.mongodb.com/cloud/atlas>`__.
+   Atlas is a cloud-hosted database-as-a-service which requires no 
+   installation, offers a free tier to get started, and provides a 
+   copyable URI to easily connect the {+intellij-short+} to your 
+   deployment.
+
+Prerequisites
+-------------
+
+Ensure that an instance of MongoDB is running and the {+intellij-short+}
+is installed.
+
+Steps
+-----
+
+.. procedure:: 
+   :style: normal 
+
+   .. step:: Show the Database tab.
+
+      To show the :guilabel:`Database` tab, click the Database Explorer 
+      icon on the far-right menu bar of IntelliJ.
+      
+      .. image:: /images/connect/intellij-database-explorer.png 
+         :alt: Click the Database Explorer icon.
+
+   .. step:: Select MongoDB as the data source.
+
+      Click the :guilabel:`+` icon, then click :guilabel:`Data Source`
+      and select :guilabel:`MongoDB` from the drop-down menu.
+
+      .. image:: /images/connect/intellij-select-data-source.png
+         :alt: Click the + button > Data Source > MongoDB.
+
+   .. step:: Enter your connection string.
+
+      Paste your :manual:`connection string 
+      </reference/connection-string/>` directly in the 
+      :guilabel:`URL` field near the bottom of the 
+      :guilabel:`Data Sources and Drivers` tab.
+      
+      Ensure that your connection string specifies the port that your
+      instance is running on. 
+      
+      Click OK.
+
+   .. step:: (*Optional*) Select databases to load.
+
+      You should see the connected MongoDB data source in Database 
+      Explorer. 
+
+      You can load as many databases as you’d like by clicking on 
+      ``1 of 11`` text as shown below:
+
+      .. image:: /images/connect/intellij-select-databases-to-load.png
+         :alt: Select the databases to load.
+
+   .. step:: (*Optional*) Launch Playgrounds.
+
+      To launch the Playgrounds, click the highlighted button in 
+      Database Explorer.
+
+      .. image:: /images/connect/intellij-launch-playgrounds.png
+         :alt: To launch the Playgrounds, click the highlighted button in Database Explorer.
+
```

Comment:
Should we include the toolbar as part of this page as well so it's becomes a cohesive flow?

Reply:
Hi @GaurabAryal! I'm not sure exactly what you're referring to here. Is this something that will be covered in https://jira.mongodb.org/browse/DOCSP-44962? Thank you!

Reply:
I'm hoping to get started [DOCSP-44962](https://jira.mongodb.org/browse/DOCSP-44962) today, just FYI :)

Anything else before I merge this one @GaurabAryal, @ajhuh-mdb? Thanks!

Reply:
all good from my end!